addon/components/discover-page/component.js

```diff
@@ -348,11 +354,6 @@ export default Ember.Component.extend(Analytics, hostAppName, {
             this.setActiveFiltersAndReload('activeFilters.providers', filter.split('OR'));
         }
     })),
-    processedTypes: Ember.computed('types', function() {
```

Comment:
Hey Henrique - This was code copied completely from SHARE.   SHARE uses it to build the options in the type facet: 
<img width="285" alt="screen shot 2017-06-13 at 10 27 46 am" src="https://user-images.githubusercontent.com/9755598/27087450-155715e0-5023-11e7-813c-10a331406405.png">.  Instead of having it in the base discover-page widget, I moved it to the widget that is using it, the search-facet-worktype facet.  This is so ASU could use it.  How it was written before, you could only really use this widget if you were using all of the standard SHARE facets as well. But since ASU is only using a selection of these facets, where it was defining which facets it wanted to use, it didn't have access to these getTypes methods, etc. to build the types. So by letting the facet itself take care of it, we open up more options to how these widgets can be combined.